client/src/App.js

```diff
@@ -35,8 +37,13 @@ library.add(
 class App extends Component {
   state = {
     userInfo: {},
-    isLogin: true,
+    isLogin: false
   };
+  componentDidMount() {
+    isAuth
```

Comment:
@AymanAlqoqa I think you have to call this function, and then since it's a network request, await it's response, for this to all work correctly:
```
async componentDidMount() {
  const isLogin = await isAuth()
  this.setState({ isLogin })
}
```

Maybe you'd want a loading state too, so the user doesn't see the app flicker from logged-out to logged-in while the network request finishes:
```
async componentDidMount() {
  this.setState({ isLoading: true })
  const isLogin = await isAuth()
  this.setState({
    isLoading: false,
    isLogin
  })
}
```